x-pack/plugins/endpoint/public/applications/endpoint/index.tsx

```diff
@@ -12,6 +12,18 @@ import { Route, Switch, Router } from 'react-router-dom';
 import { Provider } from 'react-redux';
 import { Store } from 'redux';
 import { useObservable } from 'react-use';
+import {
+  EuiPage,
```

Comment:
You don't need most of these anymore. There is now a `PageView` component in `comonents/` that you can use to display an entire page view.